src/offset/local/windows_sys/file_time.rs

```diff
@@ -0,0 +1,50 @@
+use std::{io::Error, result::Result};
```

Comment:
Is this stuff that you actually extracted from the windows-sys crate?

I think for now I'd prefer keeping all the code together in the `windows` module, which also simplifies the privacy approaches. It can really be quite minimal.

Reply:
I see you moved all the code from `windows_sys` back into a single module, but I actually meant moving everything into `windows` (just moving the `windows_sys` stuff to the bottom of the module).